plugins/skip-button-plugin/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>SkipButtonPlugin Demo</title>
+  <link rel="stylesheet" href="npm:@srgssr/pillarbox-web/dist/pillarbox.min.css"/>
+  <link rel="stylesheet" href="./scss/skip-button-plugin.scss"/>
+  <style>
+    html,
+    body {
+      height: 100%;
+    }
+
+    body {
+      margin: 0;
+      overflow: hidden;
+    }
+  </style>
+</head>
+<body>
+<video id="player" class="pillarbox-js" controls muted></video>
+
+<script type="module">
+  import pillarbox from '@srgssr/pillarbox-web';
+  import './src/skip-button-plugin.js';
+
+  // Handle URL parameters
+  const searchParams = new URLSearchParams(location.search);
+  const debug = searchParams.has('debug');
+  const ilHost = searchParams.get('ilHost') || undefined;
+  const language = searchParams.get('language');
+  const urn = searchParams.get('urn') || 'urn:rts:video:14683290';
+
+  // Create a pillarbox player instance with the SkipButtonPlugin plugin
+  window.player = pillarbox('player', {
+    debug,
+    language,
+    srgOptions: {
+      dataProviderHost: ilHost
+    },
+    plugins: { 'skipButtonPlugin': true }
```

Comment:
The great thing is that if I delete `plugins: { 'skipButtonPlugin': true }` and put `skipButton: true` in its place, it works too. Which is good, because in this case a component makes more sense than a plugin. Because there isn't really an API that a developer can use.

Reply:
I can remove the plugin entirely and force integrators to do it with `skipButton: true` if you would prefer it.

Reply:
I migrated this element from a plugin into a button. I've also added a new prompt in the `create` script to enable users to specify whether they want to generate a new component, plugin, or button.